js/web/src/components/RecentlyPurchased.js

```diff
@@ -132,8 +138,7 @@ const RecentlyPurchased = (props) => {
                     </Link>
                     <Copy sx={{ paddingLeft: 2 }}>
                       <Typography align="left" variant="h3" color="blue">
-                        {sales} Releases were sold in the last {dayDifference}{' '}
-                        days
+                        {`${sales} copies were sold in the ${dayCopy}`}
```

Comment:
"the" needs to get included w the variable day copy. current it renders as "x copies sold in the today"